test/int/clusterwide/dbuser_test.go

```diff
@@ -137,6 +138,9 @@ func buildConnectionSecret(name string) corev1.Secret {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace.Name,
+			Labels: map[string]string{
+				connectionsecret.TypeLabelKey: connectionsecret.CredLabelVal,
+			},
```

Comment:
Why was this addition needed?

Reply:
The manager on the cluster-wide integration test was built very differently from the production version. Being cluster-wide, we should be watching only labeled secrets, and with the fix, this had to be added.